lib/v2/alistapart/topic.js

```diff
@@ -0,0 +1,32 @@
+const { getData, getList } = require('./utils');
+
+module.exports = async (ctx) => {
+    const baseUrl = 'https://alistapart.com';
+    const searchRoute = '/wp-json/wp/v2/categories?search=';
+    const articleRoute = '/wp-json/wp/v2/article?categories=';
+    const { topic } = ctx.params;
+    const id = (await getData(`${baseUrl}${searchRoute}${topic}`))[0].id;
+    const data = await getData(`${baseUrl}${articleRoute}${id}`);
+    const listItems = await getList(data);
+    const items = await Promise.all(
+        listItems.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const category = await getData(`https://alistapart.com/wp-json/wp/v2/categories?post=${item.id}`);
+                item.category = category.map((v) => v.name);
+                const author = await getData(`https://alistapart.com/wp-json/wp/v2/coauthors?post=${item.id}`);
+                item.author = author.map((v) => v.name).join(', ');
+                return item;
+            })
+        )
+    );
```

Comment:
> It will be similar to https://github.com/DIYgod/RSSHub/pull/12824#discussion_r1265631483
>
> > If you want categories and tags for each post in the posts API, you can add the `_embed` search parameter.

If you add the `_embed` search parameter to the **article API**, the nominal value of author and category will returned in `_embedded` as well. You don't need to visit `/v2/categories` and `/v2/coauthors`.